### PR TITLE
022548

### PR DESCRIPTION
**Pull request by eskila**
Finished the mastermix creation protocol, which represents the majority of the work. Only sample and mastermix distribution left.

### CODE REVIEW

**Comment by eskila** [comment]
I am considering this protocol finished and ready for review.

File: protocols/022548-2/022548-2.ot2.apiv2.py [hunk 1]
```diff
@@ -0,0 +1,820 @@
...
+
+    '''
+    sample_rack_1 = ctx.load_labware(
+        sample_tuberack_loader[0], sample_tuberack_loader[1][0])
+    sample_rack_2 = ctx.load_labware(
+        sample_tuberack_loader[0], sample_tuberack_loader[1][1])
+    sample_rack_3 = ctx.load_labware(
+        sample_tuberack_loader[0], sample_tuberack_loader[1][2])
+    target_plate = ctx.load_labware(
+        target_plate_loader[0], target_plate_loader[1])
```
Comment (chazmanc) [inline protocols/022548-2/022548-2.ot2.apiv2.py:181]: this is a bit extraneous... you could create a list of sample tuberacks `sample_tuberacks = [ctx.load_labware("nest_32_tuberack_8x15ml_8x15ml_8x15ml_8x15ml", s) for s in [2, 4, 7]]`

File: protocols/022548-1/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# 022548-1 - DNA extraction: Mastermix creation
```
Comment (chazmanc) [inline protocols/022548-1/README.md:1]: Remove the hexcode and change the folder to `022548`

**Review by chazmanc — CHANGES_REQUESTED** [review]

File: protocols/022548-2/022548-2.ot2.apiv2.py (hunk 1 above)
Comment (eskila) [reply protocols/022548-2/022548-2.ot2.apiv2.py:181]: I followed your example and loaded the tuberacks with list comprehension.

File: protocols/022548-2/022548-2.ot2.apiv2.py [hunk 3]
```diff
@@ -0,0 +1,820 @@
...
+     "master_mix_range",
+     "mastermix_max_vol",
+     "mastermix_mix_rate_multiplier",
+     "mm_aspiration_flowrate_multiplier",
+     "mm_dispense_flowrate_multiplier",
+     "p300_mount",
+     "m300_mount",
+     "do_mm_resusp_pause")
+
+    is_debug_mode = False
```
[the remark this answers is not in the document]
Comment (eskila) [reply protocols/022548-2/022548-2.ot2.apiv2.py:89]: Added as a fields parameter

**Pull request merged**